usecase/garmd.go

```diff
@@ -66,8 +67,45 @@ func New(cfg config.Config) (GarmDaemon, error) {
 	}, nil
 }
 
+func NewX509(cfg config.Config) (GarmDaemon, error) {
+	pollInterval, err := time.ParseDuration(cfg.X509.PollInterval)
+	if err != nil {
+		return nil, errors.Wrap(err, "invalid 	config .x509.poll_interval ["+cfg.X509.PollInterval+"]")
+	}
+
+	certReloader, err := service.NewCertReloader(service.CertReloaderCfg{
```

Comment:
We already have the CA config. I think it is better to handle it.
https://github.com/AthenZ/garm/blob/820f972fd56b675cde0a26d29a0b9b2f393f54d3/config/config.go#L130-L131

Reply:
OK

Reply:
Handled. Please take a look for the entire code changes.

Reply:
Run `loadCa()` only during the initialization